src/components/results/shortcircuit/shortcircuit-analysis-result-table.tsx

```diff
@@ -341,53 +266,103 @@ const ShortCircuitAnalysisResultTable: FunctionComponent<
         }
     }, []);
 
-    const onFilterChanged = useCallback(
-        (e: FilterChangedEvent) => {
-            // to see what contains filter model : https://www.ag-grid.com/javascript-data-grid/filter-api/
-            const formattedFilter = Object.entries(e.api.getFilterModel()).map(
-                ([column, filter]) => {
-                    return {
-                        dataType: filter.filterType,
-                        type: filter.type,
-                        value: filter.filter,
-                        field: fromFrontColumnToBack(column),
-                    };
-                }
-            );
-
-            updateFilter(formattedFilter);
+    const getCurrent = useCallback(
+        (x: SCAFaultResult | SCAFeederResult) => {
+            let current = NaN;
+            if (analysisType === ShortCircuitAnalysisType.ALL_BUSES) {
+                current = x.current;
+            } else if (analysisType === ShortCircuitAnalysisType.ONE_BUS) {
+                current = x.positiveMagnitude;
+            }
+            return current;
         },
-        [updateFilter, fromFrontColumnToBack]
+        [analysisType]
     );
 
-    const onSortChanged = useCallback(
-        (e: SortChangedEvent) => {
-            // We filter and sort the array and only keep the fields we need
-            // The order is important, it decides in which order the columns are sorted in the back-end
-            const columnStates = e.columnApi
-                .getColumnState()
-                .filter(function (s) {
-                    return s.sort != null;
-                })
-                .sort(function (a, b) {
-                    if (a.sortIndex == null || b.sortIndex == null) {
-                        return 0;
+    const flattenResult = useCallback(
+        (shortCircuitAnalysisResult: SCAFaultResult[]) => {
+            const rows: ShortCircuitAnalysisAGGridResult[] = [];
+
+            shortCircuitAnalysisResult?.forEach(
+                (faultResult: SCAFaultResult) => {
+                    const fault = faultResult.fault;
+                    const limitViolations = faultResult.limitViolations ?? [];
+                    let firstLimitViolation;
+                    if (limitViolations.length > 0) {
+                        let lv = limitViolations[0];
+                        firstLimitViolation = {
+                            limitType: intl.formatMessage({
+                                id: lv.limitType,
+                            }),
+                            limitName: lv.limitName,
```

Comment:
Did we need limitName? If so, we need to check with POs if we should add the formatter as in AS